Extensions/editable_reblogs.js

```diff
@@ -23,44 +25,39 @@ XKit.extensions.editable_reblogs = new Object({
 	run: function() {
 		this.running = true;
 
-		XKit.interface.post_window_listener.add("editable_reblogs", XKit.extensions.editable_reblogs.post_window);
+		XKit.interface.post_window_listener.add("editable_reblogs", this.post_window.bind(this));
+
 		XKit.tools.add_css(".control-reblog-tree {display: none; } .post-form--header .reblog-title {margin: 10px; color: #444} "
 			+ ".xkit-editable-reblogs-button { float: right; margin-left: 20px; }"
 			+ ".control.right.xkit-editable-reblogs-control { display: block; width: 210px; text-align: right }", "editable_reblogs_remove_content_tree");
 
-		var pat_ver = XKit.tools.parse_version(XKit.installed.get("xkit_patches").version);
-		if (pat_ver.major <=5 && pat_ver.minor <= 3 && pat_ver.patch <= 0){
```

Comment:
We could use the version compare function that xkit_update uses for this I guess. Just pass min required version